src/components/NewClusterModal.js

```diff
@@ -977,17 +963,60 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           onDismiss
         }),
         div(['Cloud environments consist of an application, cloud compute and a persistent disk']),
-
-        renderCostBreakdown(),
-        renderApplicationSection(),
-        renderRuntimeSection(),
-        !!isPersistentDisk && renderPersistentDiskSection(),
-        // TODO PD: What do we do with this?
-        !sparkMode && !isPersistentDisk && div([
-          p(['Time to upgrade your compute runtime. Terra’s new persistent disk feature will safegard your work and data.']),
-          h(Link, { onClick: () => this.setState({ viewMode: 'aboutPersistentDisk' }) }, ['Learn more'])
-        ]),
-        bottomButtons()
+        simplifiedForm ?
+          h(Fragment, [
+            div({ style: styles.whiteBoxContainer }, [
+              div({ style: { display: 'flex', alignItems: 'flex-start', justifyContent: 'space-between' } }, [
+                div({ style: { marginRight: '2rem' } }, [
+                  div({ style: { fontSize: 16, fontWeight: 600 } }, ['Use default resource']),
+                  ul({ style: { paddingLeft: '1rem', marginBottom: 0, lineHeight: 1.5 } }, [
+                    li([
+                      div([packageLabel]),
+                      h(Link, { onClick: () => this.setState({ viewMode: 'packages' }) }, ['What’s installed on this environment?'])
+                    ]),
+                    li({ style: { marginTop: '1rem' } }, [
+                      'Default compute size of ', span({ style: { fontWeight: 600 } }, [cpu, ' CPUs']), ', ',
+                      span({ style: { fontWeight: 600 } }, [memory, ' GB memory']), ', and a ',
+                      span({ style: { fontWeight: 600 } }, [selectedPersistentDiskSize, ' GB persistent disk']), ' ',
+                      'to keep your data even after you delete your compute'
+                    ])
+                  ])
+                ]),
+                renderActionButton()
+              ]),
+              renderCostBreakdown()
+            ]),
+            div({ style: styles.whiteBoxContainer }, [
+              div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'space-between' } }, [
```

Comment:
TOL: Luck has it that `center` appears to also accomplish `baseline`. I could argue either one being the intent.

Reply:
Going to leave it as center unless we see otherwise.